.github/CODEOWNERS

```
@@ -238,15 +225,15 @@
 # Servers
 
 /servers/                                     @godotengine/_systems
-/servers/**/audio*                            @godotengine/audio
-/servers/**/camera*                           @godotengine/xr
-/servers/**/debugger*                         @godotengine/debugger
-/servers/**/display*                          @godotengine/_platforms
-/servers/**/navigation*                       @godotengine/navigation
-/servers/**/physics*                          @godotengine/physics
-/servers/**/rendering*                        @godotengine/rendering
-/servers/**/text*                             @godotengine/gui-nodes
-/servers/**/xr*                               @godotengine/xr
+/servers/**/audio_*                           @godotengine/audio
+/servers/**/camera_*                          @godotengine/xr
+/servers/**/debugger_*                        @godotengine/debugger
+/servers/**/display_*                         @godotengine/_platforms
+/servers/**/navigation_*                      @godotengine/navigation
+/servers/**/physics_*                         @godotengine/physics
+/servers/**/rendering_*                       @godotengine/rendering
+/servers/**/text_*                            @godotengine/gui-nodes
+/servers/**/xr_*                              @godotengine/xr
```

I think we need specifics for just the base level cases here, this change will make `/servers/rendering` no longer be owned, nor will `xr`, `audio`, or `text`

So we might want to restore this to the state before #97866 to start with and then redo this in a future PR, so:
```suggestion
/servers/audio*                               @godotengine/audio
/servers/camera*                              @godotengine/xr
/servers/display_server.*                     @godotengine/_platforms
/servers/navigation_server*.*                 @godotengine/navigation
/servers/physics*                             @godotengine/physics
/servers/rendering*                           @godotengine/rendering
/servers/text_server.*                        @godotengine/gui-nodes
/servers/xr*                                  @godotengine/xr
```

---

`/servers/xxx*` will not include subdirectories, so we might want two lines for each:

https://git-scm.com/docs/gitignore#_pattern_format

> An asterisk "*" matches anything except a slash.
> A trailing "/**" matches everything inside.

```
/servers/text_*                             @godotengine/gui-nodes
/servers/text/**                            @godotengine/gui-nodes
```

---

Agreed, was mainly wanting to fix this directly and then discuss more in detail the specifics in a follow-up, but if this works I'd say to go with that suggestion

---

Combining the two should work for now